vuepress/docs/docs/interop.md

```diff
@@ -111,8 +111,19 @@ The conversion rules are the following:
 - there should be an implicit `Schema` for both the input and the output types and an implicit `ArgBuilder` for the input type (see the [dedicated docs](schema.md))
 
 Let's look at an example. Imagine we have the following Tapir endpoint:
-```scala
-val addBook: Endpoint[(Book, String), Nothing, Unit, Nothing] =
+```scala mdoc:silent:reset
+import caliban._
```

Comment:
Is it mandatory to have all imports visible? It goes a little against the flow of the tutorial. How about adding needed imports in each snippet progressively?

Reply:
Makes sense. Done